packages/scandipwa/src/component/ProductGallery/ProductGallery.container.js

```diff
@@ -101,7 +101,7 @@ export class ProductGalleryContainer extends PureComponent {
             areDetailsLoaded,
             product: {
                 media_gallery_entries: mediaGallery = [],
-                [THUMBNAIL_KEY]: { url } = {},
```

Comment:
I think this modification does not do what you expect it to.
Both `IMAGE_TYPE` and `THUMBNAIL_KEY` variables have constant values here (and everywhere else), they come from the `.config.js` file. Hence, `IMAGE_TYPE || THUMBNAIL_KEY` will __always__ return `IMAGE_TYPE`.